### PR TITLE
add file for hoolie-popup task

### PR DESCRIPTION
**Pull request by maksimsergeev97**
### HTML і CSS практика: Hooli-style Popup
[Demo](https://maksimsergeev97.github.io/Hooli-style-popup) | [Code base](https://github.com/maksimsergeev97/Hooli-style-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by maksimsergeev97** [comment]
Hey! 
my code has been validated. 
I took into account all the tips from common mistakes when writing code

File: submissions/maksimsergeev97/hoolie-style-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                        <li><a class="popup__link__item" href="#"><img class="popup__icon" src="icons/search.png" alt="search icon"><span>Search</span></a></li>
+                        <li><a class="popup__link__item" href="#"><img class="popup__icon" src="icons/photos.png" alt="photo icon"><span>Photo</span></a></li>
+                        <li><a class="popup__link__item" href="#"><img class="popup__icon" src="icons/apps.png" alt="app icon"><span>Apps</span></a></li>
+                        <li><a class="popup__link__item" href="#"><img class="popup__icon" src="icons/documents.png" alt="documents icon"><span>Documents</span></a></li>
+                       
+                        <li class="more">
+                            <input id="more__button" type="checkbox" class="more__input">
+                            <label for="more__button">
+                                <span class="more__checkbox">More</span>
+                            </label>
```
Comment (A-Ostrovnyy) [inline submissions/maksimsergeev97/hoolie-style-popup/index.html:59]: Add styles on `:focus` state)

File: submissions/maksimsergeev97/hoolie-style-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                <li class="menu__list__item"><a class="menu__item__text" href="#">Mail</a></li>
+                <li class="menu__list__item"><a class="menu__item__text" href="#">Images</a></li>
+
+                <li class="menu__list__item popup">
+                    <label for="popup__button">
+                        <span class="popup__checkbox menu__item__img"></span>
+                    </label>
+                    <input id="popup__button" type="checkbox" class="popup__input">
+
+                    <ul class="popup__wrapper">
```
Comment (A-Ostrovnyy) [inline submissions/maksimsergeev97/hoolie-style-popup/index.html:41]: Fix popup position on the mobile version 
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome work!
See my comments below.

**Comment by maksimsergeev97** [comment]
Hey! Thanks for the review! Code fixed!

File: submissions/maksimsergeev97/hoolie-style-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                        <li><a class="popup__link__item" href="#"><img class="popup__icon" src="icons/calendar.png" alt="calendar icon"><span>Calendar</span></a></li>
+                        <li><a class="popup__link__item" href="#"><img class="popup__icon" src="icons/mail.png" alt="mail icon"><span>Mail</span></a></li>
+                        <li><a class="popup__link__item" href="#"><img class="popup__icon" src="icons/news.png" alt="news icon"><span>News</span></a></li>
```
Comment (A-Ostrovnyy) [inline submissions/maksimsergeev97/hoolie-style-popup/index.html:44]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/maksimsergeev97/index.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```
Comment (A-Ostrovnyy) [inline submissions/maksimsergeev97/index.html:1]: This file is redundant, also, all files must be in the task folder. Remove this file.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good prgress!

**Comment by maksimsergeev97** [comment]
Hey! Thanks for the help ! Fixed. I ask for a review, again :)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**